FIRBoundaries.dat

```diff
@@ -4444,223 +4444,571 @@ ENOB|0|0|20|63.05266|0.000000|82.000000|30.000000|72.500000|15.000000
+50.866667|24.15
+50.833333|24.083333
+50.833333|23.966667
+50.766667|23.966667
+50.766667|24.033333
+50.716667|24.033333
+50.716667|24.066667
+50.633333|24.1
+50.45|24.05
+50.416667|23.916667
+50.416667|23.783333
+50.383333|23.716667
+50.333333|23.683333
+50.283333|23.583333
+50.166667|23.416667
+50.05|23.233333
+49.533333|22.666667
+49.333333|22.8
+49.3|22.8
+49.233333|22.733333
+49.133333|22.8
+49.083333|22.9
+49.016667|22.833333
+49.05|22.683333
+49.1|22.583333
+49.1|22.433333
+49.15|22.383333
+49.15|22.283333
+49.216667|22.033333
+49.35|21.966667
+49.383333|21.833333
+49.35|21.783333
+49.45|21.633333
+49.416667|21.483333
+49.45|21.283333
+49.4|21.2
+49.433333|21.133333
+49.425278|21.066944
+49.366667|21.1
+49.3|20.933333
+49.333333|20.833333
+49.416667|20.733333
+49.409167|20.65
+49.383333|20.6
+49.416667|20.466667
+49.383333|20.4
+49.4|20.316667
+49.35|20.316667
+49.35|20.216667
+49.3|20.133333
+49.183333|20.066667
+49.222222|20.0
+49.233333|19.916667
+49.183333|19.833333
+49.216111|19.758611
+49.283333|19.816667
+49.4|19.783333
+49.383333|19.733333
+49.4|19.65
+49.416667|19.666667
+49.433333|19.616667
+49.507778|19.555556
+49.6|19.45
+49.580556|19.410833
+49.533333|19.366667
+49.533333|19.266667
+49.416667|19.2
+49.4|18.983333
+49.5|18.983333
+49.516667|18.85
+49.683333|18.816667
+49.716667|18.633333
+49.9|18.516667
+49.966667|18.216667
+50.0|18.2
+49.983333|18.166667
+50.066667|18.033333
+50.2232|18.0631
+50.2756|18.1129
+50.3134|18.1451
+50.3912|18.2442
+50.4104|18.2711
+50.4255|18.2941
+50.534|18.4418
+50.5643|18.5118
+51.0349|19.084
+51.21|19.4506
+51.2945|20.1347
+52.16575|20.967122
+52.1736|21.5551
+52.1117|23.2416
+52.1117|23.2416
+EPWW-NW|0|0|23|51.281100|14.225300|53.111800|19.241600|52.133334|16.150000
+51.483333|14.933333
+51.5|14.833333
+51.566667|14.75
+51.633333|14.766667
+51.833333|14.616667
+51.933333|14.716667
+52.083333|14.75
+52.15|14.683333
+52.25|14.7
+52.3|14.583333
+52.433333|14.566667
+52.583333|14.6
+52.5851|15.333
+53.1118|16.3437
+52.4405|17.3442
+52.403301|18.0
+52.3906|18.1023
+52.16575|20.967122
+51.5948|18.0
+51.45|17.4042
+51.4303|17.1417
+51.3703|16.0
+51.483333|14.933333
+EPWW-SW|0|0|61|51.281100|14.225300|53.111800|19.241600|50.933334|17.150000
+50.066667|18.033333
+50.033333|18.0
+50.033333|18.05
+50.0|18.05
+50.016667|18.0
+49.966667|17.866667
+50.021389|17.7875
+50.166667|17.6
+50.2|17.75
+50.266667|17.716667
+50.3|17.75
+50.316667|17.683333
+50.25|17.616667
+50.3|17.433333
+50.266667|17.366667
+50.366667|17.2
+50.433333|16.916667
+50.4|16.866667
+50.283333|17.016667
+50.216667|17.016667
+50.233333|16.95
+50.2|16.833333
+50.1|16.7
+50.122222|16.5925
+50.216667|16.55
+50.366667|16.35
+50.433333|16.2
+50.566667|16.433333
+50.65|16.333333
+50.666667|16.233333
+50.616667|16.2
+50.65|16.1
+50.6|16.033333
+50.683333|16.0
+50.666667|15.866667
+50.75|15.816667
+50.733333|15.75
+50.816667|15.45
+50.766667|15.4
+51.0|15.25
+51.016667|15.0
+51.3|15.033333
+51.416667|14.966667
+51.483333|14.933333

```

Comment:
```suggestion
```
duplicate vertex

Reply:
Seems like another error in original data... We will fix it.